tests/integrations/mcs/resourcemanager/resource_manager_test.go

```diff
@@ -940,6 +940,21 @@ func (suite *resourceManagerClientTestSuite) TestBasicResourceGroupCURD() {
 			re.Equal(1, len(groups1))
 		}
 	}
+
+	// test restart cluster
```

Comment:
I don't think this test reflects the pr fix, maybe it's best to change the default rg to ensure consistency.